---
string name
string[] type
float32 lambda1
float32 lambda2
string activation_state
float32[] weight
uint32[] indices
uint32 size
string[] disabled_joints
string lib_name
